docusaurus.config.ts

```diff
@@ -113,6 +119,7 @@ const config: Config = {
           },
           {
             label: 'Get Involved',
+            description: 'Welcome! This documentation is designed to provide you with a comprehensive understanding and step-by-step guidance for becoming a node operator within the Oasis Network.',
```

Comment:
These should go into each .md file at the top, where there is the metadata section. See https://docusaurus.io/docs/markdown-features#front-matter

Reply:
Let's move the description to the sidebars files then. Links here are just for the footer.